src/Components/__stories__/ArtworkGrid.story.tsx

```diff
@@ -19,12 +19,169 @@ function GridExample(props: { artistID: string }) {
       `}
       variables={{ artistID: props.artistID }}
       render={readyState => {
-        return readyState.props && <ArtworkGrid {...readyState.props.artist as any} />
+        return (
+          readyState.props && (
+            <RelayArtworkGrid {...readyState.props.artist as any} />
+          )
+        )
       }}
     />
   )
 }
 
-storiesOf("Components/Artworks/ArtworkGrid", module).add("A typical grid", () => {
-  return <GridExample artistID="banksy" />
-})
+storiesOf("Components/Artworks/ArtworkGrid", module)
+  .add("A typical grid", () => {
+    return <GridExample artistID="banksy" />
+  })
+  .add("Without Relay", () => {
+    return <ArtworkGrid artworks={artworks as any} useRelay={false} />
+  })
+
+const artworks = {
+  edges: [
+    {
+      node: {
```

Comment:
#minor this might be better off in its own fixture.json somewhere (just for readability).